apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -904,6 +904,24 @@ func (m *ModelManagerAPI) getModelInfo(tag names.ModelTag, withSecrets bool, wit
 		return params.ModelInfo{}, errors.Trace(err)
 	}
 
+	// If the user is a controller superuser, they are considered a model
+	// admin.
+	adminAccess := m.isAdmin
+	if !adminAccess {
+		// otherwise we do a check to see if the user has admin access to the model
+		err = m.authorizer.HasPermission(permission.AdminAccess, model.ModelTag())
+		adminAccess = err == nil
+	}
+	// admin users also have write access to the model
+	writeAccess := adminAccess
+	if !writeAccess {
+		// otherwise we do a check to see if the user has write access to the model
+		err = m.authorizer.HasPermission(permission.WriteAccess, model.ModelTag())
+		writeAccess = err == nil
+	}
+	// at this point, if the user does not have write access, they must have
+	// read access otherwise we would've returned on L892
```

Comment:
Specific line reference is very fragile, especially this far down. It's likely only a matter of days before something above this changes and invalidates this reference

Reply:
@alesstimec  just state we would have returned. Remove the line reference.

Reply:
done